codegen/kuberesource/crd.go

```diff
@@ -175,7 +177,8 @@ func CustomResourceDefinition(
 		// hashstructure of the crd spec:
 		specHash, err := hashstructure.Hash(crd.Spec, nil)
 		if err != nil {
-			panic(err)
+			contextutils.LoggerFrom(nil).DPanic(err)
```

Comment:
Given that this is in codegen, I think we might prefer to panic here. I think codegen failing loudly is preferable to potentially returning an incorrect or undesirable result